objects/Editors/script.lua

```diff
@@ -66,7 +67,7 @@ function toggleObjects()
                 guid = bagObject.guid,
                 position = contents[bagObject.guid],
                 rotation = {0,180,0},
-                callback_function = function(obj) obj.setLock(true) objectsMoved = objectsMoved + 1 end,
+                callback_function = function(obj) obj.setLock(true) objectsMoved = objectsMoved + 1 obj.setPosition(contents[bagObject.guid]) end,
```

Comment:
I'm not entirely sure why this is necessary, but without it, the element bag seems to get displaced to a randomish location either above or below the table.